docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md

```diff
@@ -36,45 +36,43 @@ Please refer to the table below for the updated status of the transition process
     </tr>
     <tr>
       <td rowspan="8">Codacy app UI</td>
-      <td>Quality dashboard</td>
-      <td>Planned</td>
+      <td><a href="#migrated-pages">Quality dashboard</a></td>
+      <td>Live</td>
       <td>-</td>
     </tr>
     <tr>
-      <td>Coverage dashboard</td>
-      <td>Planned</td>
+      <td><a href="#migrated-pages">Coverage dashboard</a></td>
+      <td>Live</td>
       <td>-</td>
     </tr>
     <tr>
-      <td>Quality pull request list page</td>
-      <td>Planned</td>
+      <td><a href="#migrated-pages">Quality pull request list page</a></td>
+      <td>Live</td>
       <td>-</td>
     </tr>
     <tr>
-      <td>Coverage pull request list page</td>
-      <td>Planned</td>
+      <td><a href="#migrated-pages">Coverage pull request list page</a></td>
+      <td>Live</td>
       <td>-</td>
     </tr>
     <tr>
-      <td>Quality pull request page</td>
+      <td><a href="#quality-pull-request-page">Quality pull request page</a></td>
       <td>Planned</td>
       <td>-</td>
     </tr>
     <tr>
-      <td>
-        <a href="#diff-tabs">Coverage pull request page</a>
-      </td>
-      <td>Live (partially migrated)</td>
+      <td><a href="#diff-tabs">Coverage pull request page</a></td>
+      <td>In progress</td>
```

Comment:
This one should be Live, right?

Reply:
Aren't we getting data from the new engine only for the Diff tab? 🤔

Reply:
Updated.